### ISSUE TITLE
[Bug]: Overriding a "standard" alias in config/app.php does not work

### ISSUE THREAD

**Issue by raveren** [laravel-idea/plugin#920]
### Bug description

```php
# config/app.php

return [
   // ...
    'aliases' => Facade::defaultAliases()->merge([
	    // ...
        'Helper' => App\Libs\Helper::class, // this works 
        'Number' => App\Libs\Number::class, // this does not
    ])->toArray(),
];
```

The generated helper code:

```php
# vendor/_laravel_idea/_ide_helper_facades.php

namespace {
    // ...
    class Helper extends App\Libs\Helper {}
    // ...
    class Number extends Illuminate\Support\Number {} // <- this should be App\Libs\Number
    // ...
}
```

I bet it's broken because `Number` is a default Laravel alias (included in `Facade::defaultAliases()`).

### Plugin version

8.0.3+

### Operating system

Windows

**Comment by adelf** [comment]
Laravel Idea also gets them from `bootstrap/cache/packages.php` file. Seems, there is an old value there, which overrides the new one. Could you try to delete it?

**Comment by raveren** [comment]
Wow, you're amazing to answer so quickly, thanks for all your hard work!!! 🙏

Unfortunately, this did not help, I (1) deleted the contents of `bootstrap/cache`, and (2) the `vendor/_laravel_idea` folder and then (3) run a random PHPUnit test just so Laravel rebuilds the cache and lastly (4) generated helper code and the `_ide_helper_facades` is again the same as in OP. I also did steps 1,2,4 with same result.

Furthermore, there's no references to `Number` (no such text) inside the files in the `bootsrap\cache` directory.

**Comment by adelf** [comment]
Found a bug. Fixed. Thank you.

**Comment by raveren** [comment]
Super cool, thank you!!! ❤

**Comment by adelf** [comment]
Fixed in 8.0.4. Thank you.
